apps/cowswap-frontend/src/common/updaters/orders/UnfillableOrdersUpdater.ts

```diff
@@ -262,6 +263,8 @@ async function _getOrderPrice(
     receiver: order.receiver,
     isEthFlow,
     priceQuality: getPriceQuality({ verifyQuote }),
+    appData: order.appData ?? undefined,
+    appDataHash: order.appDataHash ?? undefined,
```

Comment:
Bonus: pass the original appData so we can get more precise quotes.
It could have for example the permit hook, which affects the fee price.